utils/sl/check_script.py

```diff
@@ -135,17 +136,20 @@ def write_variables(outfile, prefix, variables):
                 outfile.write("%s%s = None\n" % (prefix, variable))
 
         outfile.write("# pylint: enable=invalid-name\n")
+        outfile.write("# pylint: enable=import-outside-toplevel\n")
         return newlines
 
     @staticmethod
     def write_header(outfile):
         """write the header"""
         outfile.write("""# pylint: disable=wildcard-import
+# pylint: disable=import-outside-toplevel
 from __future__ import print_function
```

Comment:
Do we still need this line now everything is python3?

Reply:
Probably not